src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -424,6 +424,34 @@ BOOST_AUTO_TEST_CASE(GradientLinearLayerTest)
   BOOST_REQUIRE_LE(CheckGradient(function), 1e-4);
 }
 
+/**
+ * Simple linear layer with only output size test.
+ */
+BOOST_AUTO_TEST_CASE(SimpleLinearLayerOutputSizeTest)
+{
+  FFN<NegativeLogLikelihood<>, ConstInitialization> modelA, modelB;
```

Comment:
Ah, I guess this is why you added the default parameter to `ConstInitialization`.  I guess, without setting that, you could do this instead:

```
FFN<NegativeLogLikelihood<>, ConstInitialization> modelA(NegativeLogLikelihood<>(), ConstInitialization(0.5));
FFN<NegativeLogLikelihood<>, ConstInitialization> modelB(NegativeLogLikelihood<>(), ConstInitialization(0.5));
```
```